src/commands/new.rs

```diff
@@ -66,6 +67,15 @@ pub struct TemplateNewCommandCore {
     /// An optional argument that allows to skip creating .gitignore
     #[clap(long = "no-vcs", takes_value = false)]
     pub no_vcs: bool,
+
+    /// If the output directory already contains files, generate the new files into
+    /// it without confirming, overwriting any existing files with the same names.
+    #[clap(
+        long = "allow-overwrite",
```

Comment:
I'm not 100% sold on the name as it can somewhat beg the question of *what* we are allowing the overwrite of, but I can't think of a better alternative so I'm fine with it as is.